resources/testdriver.js

```diff
@@ -680,6 +680,134 @@
         set_spc_transaction_mode: function(mode, context=null) {
           return window.test_driver_internal.set_spc_transaction_mode(mode, context);
         },
+
+        /**
+         * Cancels the Federated Credential Management dialog
+         *
+         * Matches the `Cancel dialog
+         * <https://fedidcg.github.io/FedCM/#webdriver-canceldialog>`_
+         * WebDriver command.
+         *
+         * @param {WindowProxy} context - Browsing context in which
+         *                                to run the call, or null for the current
+         *                                browsing context.
+         *
+         * @returns {Promise} fulfilled after the cancel request is sent, or rejected
+         *                    in case the WebDriver command errors
+         */
+        cancel_fedcm_dialog: function(context=null) {
+            return window.test_driver_internal.cancel_fedcm_dialog(context);
+        },
+
+        /**
+         * Selects an account from the Federated Credential Management dialog
+         *
+         * Matches the `Select account
+         * <https://fedidcg.github.io/FedCM/#webdriver-selectaccount>`_
+         * WebDriver command.
+         *
+         * @param {int} account_index - Index of the account to select.
```

Comment:
Judging by https://jsdoc.app/tags-param.html#names-types-and-descriptions, `int` isn't a supported type in JSDoc. This isn't TypeScript, but FWIW it seems like it's not a TS type either, see https://www.typescriptlang.org/docs/handbook/basic-types.html.

So:

```suggestion
         * @param {number} account_index - Index of the account to select.
```

There isn't any type checking in testdriver.js generally, so I think we'll rely on the WebDriver endpoint returning an error if the input isn't right.

Reply:
Ah fixed, thank you!